client/hack/README.md

```diff
@@ -95,3 +99,68 @@ Update the restoreSize property to use type string only:
 
 ```
 
+* Because we need to serve both v1 and v1beta1 snapshot APIs, we need to make sure that both v1 and v1beta1 APIs are in the manifest yaml file. Because `update-crd.sh` only generates v1 manifest, make sure to copy the v1beta1 manifest below the v1 manifest after running `update-crd.sh` in the manifest yaml files. See `snapshot.storage.k8s.io_volumesnapshots.yaml` as an example. `served` is true for both v1beta1 and v1. `storage` is true for v1beta and false for v1.
```

Comment:
the "oneOf" sections in v1 is manually added, will those be removed when run controller-gen? if so, we will need to document that.

Reply:
Yes, we also have doc for the "oneOf".